pkg/kubelet/kubelet.go

```diff
@@ -450,7 +450,7 @@ func NewMainKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 	nodeIndexer := cache.NewIndexer(cache.MetaNamespaceKeyFunc, cache.Indexers{})
 	if kubeDeps.HeartbeatClient != nil {
 		fieldSelector := fields.Set{api.ObjectNameField: string(nodeName)}.AsSelector()
-		nodeLW := cache.NewListWatchFromClient(kubeDeps.HeartbeatClient.CoreV1(), "nodes", metav1.NamespaceAll, fieldSelector)
+		nodeLW := cache.NewListWatchFromClientWithMultiTenancy(kubeDeps.HeartbeatClient.CoreV1(), "nodes", metav1.NamespaceAll, fieldSelector, metav1.TenantNone)
```

Comment:
NamespaceAll but TenantNone. This combination is very weird

Reply:
this is cherry-picked from the master branch. Qian, can you comment on this ?

Reply:
I agree. It is better to use "metav1.NamespaceNone" instead of "metav1.NamespaceAll". They are both "" under the hood, but "metav1.NamespaceNone" makes more sense. 

Feel free to make the changes.

Reply:
good. in this, i would prefer to fix. 
just pushed a name change to use NamespaceNone here.